Sources/Paginator/Paginatable/RawSQLBuilderPaginatable.swift

```diff
@@ -34,6 +34,18 @@ public class RawSQLBuilder<Database, Result> where
         
         self.sqlRawCountBuilder = connection.raw(countQuery)
     }
+    
+    public func bind(_ encodable: Encodable) -> Self {
+        _ = sqlRawBuilder.bind(encodable)
```

Comment:
Can the `_ =` be avoided? Otherwise LGTM

Reply:
I just looked at it and it's just because bind on SQLRawBuilder is defined as `public func bind(_ encodable: Encodable) -> Self {`. It could be avoided like this:
```swift
    public func bind(_ encodable: Encodable) -> Self {
        sqlRawBuilder.binds.append(encodable)
        sqlRawCountBuilder?.binds.append(encodable)
        return self
    }
```
But that's really doing the same thing. So 👍 from me too 🙂